### DISCUSSION TITLE
How to custom background color ?

### DISCUSSION THREAD

**Discussion by allworldg** [projekt0n/github-nvim-theme#236] [Q&A] [answered]
I use `vim.cmd[[highlight Normal guibg = #F9F9F9]]` to custom background color . It is useful. But when I move my cursor into the lsp hover float window, the whole background color change back to white . Then I disable this colorscheme, everything is right. I guess if I can custom color by using this theme config , maybe the problem can be solved .
Love your theme , Thank you .
![image](https://user-images.githubusercontent.com/48782874/223966127-ec53c09b-f466-4d49-8838-c59fa50a3e60.png)
![image](https://user-images.githubusercontent.com/48782874/223966174-d7fb39ab-a176-4770-9d58-5d9a6c1a9b25.png)

**Comment by allworldg** [comment, marked as answer]
I have found it is a highlight buffer problem . I solve it . But I still want to know is there a config to set background color(not  theme_style).

**Reply by ful1e5** [reply]
You can override colors with the `overrides`  config in `setup`
```lua
require('github-theme').setup({
  -- ...
  overrides = function(c)
    return {
      Normal = { bg = '#000000', fg = '#ffffff' }, -- with Hex
      -- Normal = { bg = c.bg2, fg = c.fg },  -- Or with colors
    }
  end,
})

```

**Reply by allworldg** [reply]
Thank you.
